cloud/helm-chart/src/main/helm/zilla/templates/ingress.yaml

```
@@ -0,0 +1,76 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+{{ if .Values.ingress.enabled -}}
+{{- $fullName := include "zilla.fullname" . -}}
+{{- $svcPort := .Values.ingress.port -}}
```

There is also a slight syntax issue on the first line so maybe there was a copy past mixup
```suggestion
{{- if .Values.ingress.enabled -}}
{{- $fullName := include "zilla.fullname" . -}}
{{- $svcPort := .Values.service.port -}}
```

---

discussed on slack